### ISSUE TITLE
MU-GAN outputs

### ISSUE THREAD

**Issue by listener17** [mindslab-ai/nuwave#1]
Hi!

Congrats for nice results!
I looked at your MU-GAN outputs. Are these the outputs from your own implementation? 

Why didn't you try NU-GAN?
https://arxiv.org/abs/2010.11362

**Comment by junjun3518** [comment]
Thanks for the first issue!

We self-implemented MU-GAN, while specific channels are not mentioned in the MU-GAN paper, we tested several channels for stable GAN training.

For NU-GAN, we did not use it for two reasons.
First, we want the "waveform-to-waveform" method to compare with our model. NU-GAN uses STFT magnitude as input. (It's why MU-GAN was applied)
Second, while the Transformer model for audio data needs lots of details, such as positional encoding, input length (need to be fixed if sinusoidal pos enc is applied), number of attention heads, etc, the NU-GAN paper did not provide these details.
